R/session/vsc.R

```diff
@@ -28,6 +28,7 @@ load_settings <- function() {
     vsc.object_timeout = session$objectTimeout,
     vsc.globalenv = session$watchGlobalEnvironment,
     vsc.plot = setting(session$viewers$viewColumn$plot, Disable = FALSE),
+    vsc.dev.args = c(width = plot$devArgs$width, height = plot$devArgs$height),
```

Comment:
The option is used as following:

```r
dev_args <- getOption("vsc.dev.args")
do.call(png, c(list(filename = plot_file), dev_args))
```

Maybe it makes more sense to just keep it as a list.